libraries/utilities/generate_clusters_from_pool.py

```diff
@@ -363,6 +363,22 @@ def write_config(config, pool_file, use_docker):
         f.write("cbs_ssl_enabled=False\n")
         f.write("xattrs_enabled=False\n")
 
+        if sg_windows:
+            f.write("\n\n[sync_gateways:vars]\n")
+            f.write("ansible_user=Administrator\n")
+            f.write("ansible_password=Membase123\n")
+            f.write("ansible_port=5986\n")
+            f.write("ansible_connection=winrm\n")
+            f.write("ansible_winrm_server_cert_validation=ignore\n")
+
+        if sg_accel_windows:
+            f.write("\n\n[sg_accels:vars]\n")
+            f.write("ansible_user=Administrator\n")
+            f.write("ansible_password=Membase123\n")
+            f.write("ansible_port=5986\n")
+            f.write("ansible_connection=winrm\n")
+            f.write("ansible_winrm_server_cert_validation=ignore\n")
```

Comment:
Just so I'm clear, the config generation with append these values to the ansible config and assume that all of the sg / sg_accels are windows?